pkg/network/prometheus.go

```diff
@@ -72,7 +64,7 @@ func init() {
 	prometheus.MustRegister(
 		estimatedNetworkSize,
 		peersConnected,
-		neogoVersion,
+		network.NeogoVersion,
```

Comment:
As far as metric registration code. The metric itself should be registered inside `cli/server`. Our main goal is not to show this metric for NeoFS node/Morph node.